.travis.yml

```diff
@@ -1,36 +1,63 @@
 language: node_js
 matrix:
   fast_finish: true
-  # Generally using active LTS versions here, see https://github.com/nodejs/Release
+  # Generally using active LTS versions here, see https://github.com/nodejs/Release 
   include:
-    - name: "Test client"
-      node_js: 12
-      env: RUN="npm run test:client"
-    - name: "Test server"
-      node_js: 12
-      env: RUN="npm run test:server"
-    - name: "Lint and try building"
-      node_js: 12
-      env: RUN="npx concurrently --kill-others-on-fail 'npm:lint' 'npm:build'"
+  - name: Test client
+    node_js: 12
+    env: RUN="npm run test:client"
+  - name: Test server
+    node_js: 12
+    env: RUN="npm run test:server"
+  - name: Lint and build
+    node_js: 12
+    env: RUN="npx concurrently --kill-others-on-fail 'npm:lint' 'npm:build:tarball'" DEPLOY="true"
```

Comment:
Do we need `NODE_ENV=production` here since we're building the bundle for deploying?

Reply:
`NODE_ENV=production` is set in the "build" script in package.json.

And checking https://dev2.trustroots.org/assets/main.js it looks pretty minified.